bigquery/README.md

```diff
@@ -2,7 +2,18 @@
 
 This notifier pushes build data to a BigQuery instance.
 
-*Alpha Feature - Under Development*
+This notifier runs as a container via Google Cloud Run and responds to
+events that Cloud Build publishes via its
+[Pub/Sub topic](https://cloud.google.com/cloud-build/docs/send-build-notifications).
+
+For detailed instructions on setting up this notifier,
+see [Configuring BigQuery notifications](TODO).
```

Comment:
Do you mean to wait for @navierula 's docs?

Reply:
yep!